linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -386,13 +386,13 @@ static int threadpool_work_func(void* param);
 
 **SRS_THREADPOOL_LINUX_07_084: [** If the work item function is not `NULL`, `threadpool_work_func` shall execute it with `work_function_ctx`. **]**
 
-**S_R_S_THREADPOOL_LINUX_05_032: [** If the `threadpool_destroy_work_item_ptr` is not `NULL` then: **]**
+- **SRS_THREADPOOL_LINUX_05_036: [** If the `threadpool_destroy_work_item_ptr` is not `NULL` then: **]**
 
-- **S_R_S_THREADPOOL_LINUX_07_076: [** `threadpool_work_func` shall acquire the shared SRW lock by calling `srw_lock_acquire_shared`. **]**
+    - **S_R_S_THREADPOOL_LINUX_07_076: [** `threadpool_work_func` shall acquire the shared SRW lock by calling `srw_lock_acquire_shared`. **]**
 
-- **S_R_S_THREADPOOL_LINUX_05_033: [** `threadpool_work_func` shall decrement the dereferenced `threadpool_destroy_work_item_ptr`. **]**
+    - **SRS_THREADPOOL_LINUX_05_037: [** `threadpool_work_func` shall decrement the `pending_work_item_count_ptr` by calling `interlocked_decrement`. **]**
 
-- **S_R_S_THREADPOOL_LINUX_07_083: [** `threadpool_work_func` shall release the shared SRW lock by calling `srw_lock_release_shared`. **]**
+    - **S_R_S_THREADPOOL_LINUX_07_083: [** `threadpool_work_func` shall release the shared SRW lock by calling `srw_lock_release_shared`. **]**
```

Comment:
>S_R_S [](http://example.com/codeflow?start=8&length=5)

SRS

Reply:
07_083 is already used, if I do SRS then build fails marking it as a duplicate. This is reused to enhance readability. Do you know how to correctly reuse the tag? Or should it be a new one with my devid 05?